### PR TITLE
add script for creating tables

### PR DESCRIPTION
**Pull request by JK8789**
script for creating tables

### CODE REVIEW

File: Pets_script.sql [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+CREATE SCHEMA IF NOT EXISTS `mydb` DEFAULT CHARACTER SET utf8 ;
+USE `mydb` ;
+
+-- -----------------------------------------------------
+-- Table `mydb`.`persons`
+-- -----------------------------------------------------
+DROP TABLE IF EXISTS `mydb`.`persons` ;
+
+CREATE TABLE IF NOT EXISTS `mydb`.`persons` (
+  `idpersons` INT NOT NULL,
```
Comment (DanilaVaratyntsev) [inline Pets_script.sql]: Why not simple `id`? `persons` is already in the table name

File: Pets_script.sql [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+CREATE TABLE IF NOT EXISTS `mydb`.`pets` (
+  `idpets` INT NOT NULL,
+  `name` VARCHAR(45) NOT NULL,
+  `persons_idpersons` INT NOT NULL,
+  PRIMARY KEY (`idpets`, `persons_idpersons`),
+  CONSTRAINT `fk_pets_persons1`
+    FOREIGN KEY (`persons_idpersons`)
+    REFERENCES `mydb`.`persons` (`idpersons`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
```
Comment (DanilaVaratyntsev) [inline Pets_script.sql]: Please take look at other `ON DELETE` options, one of them may be more appropriate here

**Review by DanilaVaratyntsev — APPROVED** [review]

File: Pets_script.sql (hunk 1 above)
Comment (JK8789) [reply Pets_script.sql]: Yes, thank you. I fixed it

File: Pets_script.sql (hunk 2 above)
Comment (JK8789) [reply Pets_script.sql]: I think "ON DELETE SET NULL" is better for pets table,  "ON DELETE CASCADE" is better for schedule table and "ON DELETE CASCADE" is better for schedule_has_pets table.

**Pull request merged**